src/deepsparse/v2/operators/engine_operator.py

```diff
@@ -13,14 +13,14 @@
 # limitations under the License.
```

Comment:
General comment: we should also add continuous batching to the e2e testing suite. This is why I'd first land the e2e tests PR ASAP, rebase this PR, and add tests as a part of this PR (optionally, we can add a feature branch, so that this can get reviewed and landed quickly to a feature branch; on top of that feature branch we add tests)